### DISCUSSION TITLE
feature request: Scroll-to-Zoom Option

### DISCUSSION THREAD

**Discussion by sean1832** [excalidraw/excalidraw#6087] [General]
I wanted to suggest the possibility of adding a scroll-to-zoom option for 2D navigation in the canvas. This type of mechanism is already implemented in other applications like Miro, Conceptboard, and PureRef and I believe it could be a useful addition to Excalidraw.

While the current scrollwheel functionality for scrolling the canvas up and down is useful, I think offering the option for users to switch to a zoom mode could enhance the usability and user experience of the software.

I hope this suggestion is something that can be considered for future updates. Thank you for considering my thoughts.

**Comment by sojusnik** [comment]
Already suggested here: https://github.com/excalidraw/excalidraw/discussions/5515

**Reply by sean1832** [reply]
Hi, Thank you for mentioning that. My bad that I did not find it before posting the question. This issue has bogged me down for quite some time, and I wish to make some voice to it.
